src/app/public/modules/confirm/confirm-closed-event-args.ts

```diff
@@ -1,3 +1,7 @@
 export interface SkyConfirmCloseEventArgs {
+  /**
+   * Specifies an identifier to return when users select the button to close the
```

Comment:
Looks like we copied over the wrong description. This description is for the `action` property for `SkyConfirmButtonConfig`. What we need here is the description for the `action` property for `SkyConfirmCloseEventArgs`, which is in the table at https://developer.blackbaud.com/skyux/components/confirm#skyconfirminstance-events.